libs/features/pages/contact-page/src/lib/components/contact-page-form-section/contact-page-form-section.component.scss

```diff
@@ -0,0 +1,206 @@
+.contact-form {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: flex-start;
+  
+  .contact-text {
+    flex: 1.9;
+    margin-left: 65px;
+    margin-top: 60px;
+
+    h2 {
+      font-size: 35px;
+    }
+
+    p {
+      margin: 10px 0;
+    }
+  }
+  
+  form {
+    margin-right: 500px;
+    margin-top: 60px;
+  }
+  
+  label {
+    display: block;
+    margin-bottom: 1px;
+  }
+  
+  input[type="text"],
+  input[type="email"],
+  textarea,
+  select {
+    width: 200%;
+    padding: 2px;
+    margin-bottom: 20px;
+    border: none;
+    border-bottom: 2px solid #ccc;
+    background-color: transparent;
+    transition: border-bottom 0.3s ease;
+  }
+
+  input[type="text"]:focus,
+  input[type="email"]:focus,
+  textarea:focus,
+  select:focus {
+    border-bottom: 2px solid var(--elewa-website-contact-focus-color); 
+    outline: none;
+  }
+
+  #selection {
+    font-size: 15px;
+    padding: 2px;
+  }
+
+  .privacy-policy {
+    margin-top: 20px;
+    font-size: 12px;
+  }
+
+  .social-icons {
+    flex: 1;
+    display: flex;
+    flex-direction: row; 
+    align-items: flex-start;
+
+    a {
+      font-size: 24px;
+      margin-top: 418px;
+      color: var(--elewa-website-color-primary-black);
+      margin-right: 310px; 
+      margin-left: -280px;
+      text-decoration: none;
+      cursor: pointer;
+    }    
+  }
+  
+  .send-slider-button {
+    position: absolute;
+    top: 495px;
+    right: 220px; 
+  }
+}
+/**Tablet media queries **/
+@media (min-width: 481px) and (max-width: 768px) {
+  .contact-form {
+    flex-direction: row;
+    align-items: center;
+
```

Comment:
To ensure the 'contact text' section of the contact form remains at the top, the align items property should be set to flex-start